shared/settings/email/index.desktop.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import React from 'react'
+import {globalStyles, globalColors, globalMargins} from '../../styles'
+import {Box, Button, Icon, Input, Text} from '../../common-adapters'
+
+import type {Props} from './index'
+
+function VerifiedText ({isVerified, style}: {isVerified: boolean, style?: Object}) {
+  const color = isVerified ? globalColors.green2 : globalColors.red
+  return (
+    <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', alignSelf: 'center', ...style}}>
+      <Icon
+        type={isVerified ? 'iconfont-check' : 'iconfont-close'}
+        style={{color, marginRight: 3, marginTop: 2}} />
+      <Text type='BodySmall' style={{color}}>
+        {isVerified ? 'Verified' : 'Not verified'}
+      </Text>
+    </Box>
+  )
+}
+
+function UpdateEmail (props: Props) {
+  return (
+    <Box style={globalStyles.flexBoxColumn}>
+      <Input
+        floatingLabelText='Email'
+        value={props.email}
+        onChangeText={props.onChangeEmail}
+        textStyle={{height: undefined}} />
```

Comment:
it probably isn't needed anymore, but at one point text had a set height and it made aligning things harder. Let me check now